apis/r/R/QueryCondition.R

```diff
@@ -301,20 +307,24 @@ tiledbsoma_empty_query_condition <- function(somactx) {
   return(invisible(query_condition))
 }
 
-# ================================================================
 #' Initialize a 'tiledbsoma_query_condition' object
 #'
-#' Initializes (and possibly allocates) a query condition object using a triplet of
-#' attribute name, comparison value, and operator.  Six types of conditions are supported,
-#' they all take a single scalar comparison argument and attribute to compare against.
-#' At present only integer or numeric attribute comparisons are implemented.
+#' Initializes (and possibly allocates) a query condition object using a
+#' triplet of attribute name, comparison value, and operator.  Six types of
+#' conditions are supported, they all take a single scalar comparison argument
+#' and attribute to compare against. At present only integer or numeric
+#' attribute comparisons are implemented.
+#'
 #' @param attr_name A character value with the scheme attribute name
 #' @param value A scalar value that the attribute is compared against
-#' @param arrow_type_name A character value with the TileDB data type of the attribute column, for
-#' example 'float' or 'int32'
-#' @param op_name A character value with the comparison operation. This must be one of
-#' 'LT', 'LE', 'GT', 'GE', 'EQ', 'NE'.
-#' @param qc A 'tiledbsoma_query_condition' object to be initialized by this call.
+#' @param arrow_type_name A character value with the TileDB data type of the
+#' attribute column, for example 'float' or 'int32'
+#'
+#' @param op_name A character value with the comparison operation. This must be
+#' one of 'LT', 'LE', 'GT', 'GE', 'EQ', 'NE'
```

Comment:
```suggestion
#' one of 'LT', 'LE', 'GT', 'GE', 'EQ', 'NE'.
```

Reply:
And if I missed any other spots where you're not ending sentences with periods (on this or any of your other PRs today) -- my apologies for not having ticked every one, but, it is my preference that we end sentences with periods.

Reply:
If this preference of mine is contrary to accept R style, please do let me know ...

Reply:
Thank you! :)